include/crm/crm.h

```diff
@@ -29,6 +29,21 @@ extern "C" {
 
 #  include <libxml/tree.h>
 
+#ifndef PCMK_ALLOW_DEPRECATED
+/*!
+ * \brief Allow use of deprecated Pacemaker APIs
+ *
+ * By default, external code using Pacemaker headers is allowed to use
+ * deprecated Pacemaker APIs. If PCMK_ALLOW_DEPRECATED is defined to 0 before
+ * including any Pacemaker headers, deprecated APIs will be unusable. It is
+ * strongly recommended to leave this unset for production and release builds,
+ * to avoid breakage when Pacemaker upgrades deprecate more APIs. This should be
```

Comment:
I think you're missing a word between "upgrades" and "deprecate".  Other than that, fine to merge.

Reply:
reworded, thanks